From: Adrian Knoth <adi@drcomp.erfurt.thur.de>
Forwarded: not-needed
Origin: https://github.com/jackaudio/jack2/commit/4d3595ebe57e7d7b94e6e1f172edc01415129ab4
Applied-Upstream: 1.9.9.6
Description: Fix FTBFS in bleeding-edge upstream code
--- a/posix/JackPosixServerLaunch.cpp
+++ b/posix/JackPosixServerLaunch.cpp
@@ -109,7 +109,7 @@
     if (fp) {
         arguments[0] = '\0';
         res = fscanf(fp, "%s", buffer);
-        while (ret != 0 && ret != EOF) {
+        while (res != 0 && res != EOF) {
             strcat(arguments, buffer);
             strcat(arguments, " ");
             res = fscanf(fp, "%s", buffer);
